trino-aws-proxy/src/test/java/io/trino/aws/proxy/server/rest/TestProxiedErrorResponses.java

```diff
@@ -135,7 +144,13 @@ private static class HttpErrorResponseServlet
         protected void doGet(HttpServletRequest req, HttpServletResponse resp)
                 throws IOException
         {
-            String path = req.getPathInfo();
+            System.err.println("=========================");
+            System.err.println("===== HttpServletRequest =====");
+            System.err.println("=========================");
+            System.err.println("RequestURI: " + req.getRequestURI());
+            System.err.println("PathInfo: " + req.getPathInfo());
+
+            String path = firstNonNull(req.getRequestURI(), req.getPathInfo());
```

Comment:
I'm not sure familiar with the ServletAPI, but fro reading https://stackoverflow.com/questions/4931323/whats-the-difference-between-getrequesturi-and-getpathinfo-methods-in-httpservl I have a few questions.

With Airlift, the servlet is bound at the root path, right?